src/tool/confiscan.sh

```diff
@@ -115,6 +115,30 @@ else
     mkdir -p "${output_dir}"
 fi
 
+####################
+# Operating System #
+####################
+
+info "OS Information:"
+printf 'Hostname,OperatingSystem,Name,Version,Version_ID,Codename,Kernel\n' > \
+    "${output_dir}/os_info.csv"
+printf '%s,%s,%s,%s,%s,%s,%s\n' \
+    "${HOSTNAME}" \
+    "$(uname -o)" \
+    "${PRETTY_NAME}" \
+    "${VERSION_ID}" \
+    "${VERSION}" \
+    "${VERSION_CODENAME}" \
+    "$(uname -r)" >> \
+    "${output_dir}/os_info.csv"
+
+column -s, -t "${output_dir}/os_info.csv"
+
+info "Boot parameters:"
+tr ' ' '\n' < /proc/cmdline | tee -a "${output_dir}/boot_parameters.txt"
+
+column -s, -t "${output_dir}/mount_points.csv"
```

Comment:
Either I'm blind or I'm missing the part where you're gathering this info :smile:

Reply:
I think your blind 🙃. All boot params are present in the `/proc/cmdline` file, this is directed to `tr`, which is then appended to the `boot_parameters.txt` file using `tee`.

Reply:
I was referring to line 140 though, which handles uses mount_points.csv which I guess belongs in PR #18